docs/configuration_options.rst

```diff
@@ -178,7 +178,7 @@ Connector
   * Importance: medium
 
 ``partitioner.class``
-  The partitioner to use when writing data to HDFS. You can use ``DefaultPartitioner``, which preserves the Kafka partitions; ``FieldPartitioner``, which partitions the data to different directories according to the value of the partitioning field specified in ``partition.field.name``; ``TimebasedPartitioner``, which partitions data according to the time ingested to HDFS.
+  The partitioner to use when writing data to HDFS. You can use ``DefaultPartitioner``, which preserves the Kafka partitions; ``FieldPartitioner``, which partitions the data to different directories according to the value of the partitioning field specified in ``partition.field.name``; ``TimeBasedPartitioner``, which partitions data according to the time ingested to HDFS.
```

Comment:
This file is actually generated from the config class, so there are probably corresponding changes that should be made in `HdfsSinkConnectorConfig` (or the next time someone runs the config class's main method to generate this, they may revert these changes).